### PR TITLE
Improve comments

### PR DESCRIPTION
**Pull request by fsmiamoto**
While reading the source code, I saw the opportunity to improve some of the comments to make them clearer and fix some typos.

Let me know what you think and thanks for the awesome project.

### CODE REVIEW

File: fuzzyfinder.go [hunk 1]
```diff
@@ -604,16 +604,16 @@ func (f *finder) find(slice interface{}, itemFunc func(i int) string, opts []Opt
 	}
 }
 
-// Find displays a UI that provides fuzzy finding against to the passed slice.
-// The argument slice must be a slice type. If it is not a slice, Find returns
+// Find displays an UI that provides fuzzy finding against the provided slice.
```
Comment (ktr0731) [inline fuzzyfinder.go:607]: > an UI 

Should we use `a` instead of `an`?

**Review by ktr0731 — COMMENTED** [review]
@fsmiamoto 
Thanks for the great work! I'm not a native English speaker, so this kind of PR is very helpful.
I commented just one point, can you check it?

File: fuzzyfinder.go (hunk 1 above)
Comment (fsmiamoto) [reply fuzzyfinder.go:607]: Good catch!
I didn't put much thought to it, but looks like `a` is the way to go here.

**Comment by fsmiamoto** [comment]
@ktr0731 
Great!
Happy to help improve things.

**Review by ktr0731 — APPROVED** [review]
@fsmiamoto 
LGTM. Thanks for your contribution!

**Pull request merged**